values/strings.xml

```diff
@@ -755,9 +756,9 @@
     <string name="draft_alert_swat00_text">If there\'s a riot only SWAT teams can hande it.</string>
     <string name="control_continue">Continue</string>
     <string name="settings_winter">Winter</string>
-    <string name="notify_riot_reason">There\'s a riot because of %1$s!</string>
-    <string name="notify_riot_noreason">There\'s a riot for no reason!</string>
-    <string name="notify_demonstration">There\'s a demonstration in our town because of %1s! Call the police or inprove happiness, or something bad will happen!</string>
+    <string name="notify_riot_reason">There\'s a riot because of %1$s! Call SWAT teams to stop them, and improve happiness!</string>
+    <string name="notify_riot_noreason">There\'s a riot for no reason! Call SWAT teams to stop them!</string>
+    <string name="notify_demonstration">There\'s a protest in your city because of %1s! Call the police or inprove happiness, or a riot will happen!</string>
```

Comment:
```suggestion
    <string name="notify_demonstration">There\'s a demonstration in your city because of %1s! Call the police or improve happiness, or a riot will happen!</string>
```